src/api/tests/usersRoute.spec.ts

```diff
@@ -30,15 +30,5 @@ describe('Users API tests', () => {
       expect(res.statusCode).toBe(200);
       expect(sendInviteEmail).toHaveBeenCalled();
     });
-
-    test('Should deny access to an authenticated user without permission', async () => {
```

Comment:
I think this is related to my other comment - why do you think this test scenario isn't valid now?